tests/container_disk_test.go

```diff
@@ -128,6 +131,66 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				}
 			})
 		})
+		Context("[Serial]should obey the disk verification limits in the KubeVirt CR", func() {
+			AfterEach(func() {
+				// Restoring the diskVerificaton memory usage limit
+				kv := util.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.DeveloperConfiguration.DiskVerification = nil
+				_, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(kv)
+				Expect(err).To(BeNil())
+			})
+
+			It("disk verification should fail when the memory limit is too low", func() {
+				By("Reducing the diskVerificaton memory usage limit")
+				kv := util.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.DeveloperConfiguration = &v1.DeveloperConfiguration{
+					DiskVerification: &v1.DiskVerification{
+						MemoryLimit: resource.NewScaledQuantity(42, resource.Kilo),
+					},
+				}
+				_, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(kv)
+				Expect(err).To(BeNil())
+
+				By("Starting the VirtualMachineInstance")
+				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				_, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+				Expect(err).To(BeNil())
+				By("Checking that the VMI failed")
+				Eventually(func() bool {
+					vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					for _, condition := range vmi.Status.Conditions {
+						if condition.Type == v1.VirtualMachineInstanceSynchronized && condition.Status == k8sv1.ConditionFalse {
+							return true
+						}
+					}
+					return false
+				}, 3*time.Minute, 2*time.Second).Should(BeTrue())
+			})
+
+			It("disk verification should succeed when the memory limit is set to the default", func() {
```

Comment:
I think this is not necessary. There is not special image used which would justify testing it compared to the few hundred other runs where we do the verification.

Reply:
Makes sense, rolling back to 1 new test then.